### PR TITLE
Fix name for nixFlakes package

### PR DESCRIPTION
**Pull request by nre-ableton**
- Fix name for nixFlakes package
- Version 0.0.4

### CODE REVIEW

**Comment by ala-ableton** [comment]
@nre-ableton I'm interested in reviewing this PR, thanks!

**Comment by nre-ableton** [comment]
@ala-ableton Please go ahead, then. I would like to get this PR merged ASAP so we can start test-staging-jobs right away.

**Comment by mst-ableton** [comment]
lgtm. Looks like the alias has been around for [quite a long time](https://github.com/NixOS/nixpkgs/commit/a1be7cb1c3efe7006984fb15a72afc83889ea356#diff-29ed0f5fda84c91253503c1664d60a194a324e29518d472adf846f30b8db52e3R666), but the throw [only got added recently](https://github.com/NixOS/nixpkgs/commit/d06207386df9a53fe01f8a30130dfc9a839fc8fc#diff-29ed0f5fda84c91253503c1664d60a194a324e29518d472adf846f30b8db52e3R778).

**Review by mst-ableton — APPROVED** [review]
lgtm. Looks like the alias has been around for [quite a long time](https://github.com/NixOS/nixpkgs/commit/a1be7cb1c3efe7006984fb15a72afc83889ea356#diff-29ed0f5fda84c91253503c1664d60a194a324e29518d472adf846f30b8db52e3R666), but the throw [only got added recently](https://github.com/NixOS/nixpkgs/commit/d06207386df9a53fe01f8a30130dfc9a839fc8fc#diff-29ed0f5fda84c91253503c1664d60a194a324e29518d472adf846f30b8db52e3R778).

File: tasks/main.yml [hunk 1]
```diff
@@ -71,7 +71,7 @@
       # We need to use shell instead of command because of the above --login flag, since
       # nix-env isn't in the default path.
       # noqa command-instead-of-shell
-      ansible.builtin.shell: "nix-env -iA nixpkgs.nixFlakes"
+      ansible.builtin.shell: "nix-env -iA nixpkgs.nixVersions.stable"
```
Comment (ala-ableton) [inline tasks/main.yml:74]: The task still mentions "Nix Flakes package"

Comment (nre-ableton) [reply tasks/main.yml:74]: It's an alias, so isn't this technically still correct?

Comment (ala-ableton) [reply tasks/main.yml:74]: Quoting https://github.com/NixOS/nixpkgs/pull/123898

> There no longer is a separate `nixFlakes` package, flake support has been in master for a while.

What we are installing is the Nix stable package.

Comment (nre-ableton) [reply tasks/main.yml:74]: Fair enough! Fixed in 5d52dc6.

**Review by ala-ableton — APPROVED** [review]

**Comment by nre-ableton** [comment]
@ablbot autosquash

**Pull request merged**